build/config.props

```diff
@@ -19,7 +19,7 @@
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.2</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.3</ReleaseLabel>
```

Comment:
why not increment the assembly build number?

Reply:
insertion hasn't happened yet, and I incremented the assembly number after the last insertion.